composites/Plugin/SnippetEditor/components/SnippetEditor.js

```diff
@@ -37,6 +37,8 @@ const CloseEditorButton = SnippetEditorButton.extend`
 	margin-left: 20px;
 `;
 
+const MetaDescriptionPlaceholder = "Modify your meta description by editing it right here";
```

Comment:
Why not put this as the default in defaultProps? That way people can override it without editing this file.